### PR TITLE
Step 1: Create the Django model Define a Django model named 'Monitor' with fields such as 'url', 'keyword', 'last_checked_time', 'status', and 'user'. #1897

### PR DESCRIPTION
**Pull request by Sarthak5598**
#1897

### CODE REVIEW

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
Can you also add the created and modified fields as well as the migration and the class for the admin?

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
Please also add the form to create an entry as well as the created / modified field names.

**Comment by Sarthak5598** [comment]
sorry, I forgot about this, I will make the changes !!

**Comment by Sarthak5598** [comment]
image

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
Almost there we also need a created and modified field. Please see the other table for an example.

File: website/views.py [hunk 1]
```diff
@@ -2438,6 +2445,19 @@ def create_wallet(request):
...
+    if request.method == "POST":
+        form = MonitorForm(request.POST)
+        if form.is_valid():
+            monitor = form.save(commit=False)
+            monitor.user = request.user  # Assuming you have a logged-in user
+            monitor.save()
+            # Redirect to a success page or render a success message
+    else:
+        form = MonitorForm()
+    return render(request, "Moniter.html", {"form": form})
```
Comment (arkid15r) [inline website/views.py:2458]:
```suggestion
    return render(request, "monitor.html", {"form": form})
```

Comment (arkid15r) [inline website/views.py:2458]: Do you use this view anywhere?

File: website/views.py [hunk 2]
```diff
@@ -4257,3 +4277,15 @@ def get_context_data(self, **kwargs):
...
+def create_monitor(request):
+    if request.method == "POST":
+        form = MonitorForm(request.POST)
+        if form.is_valid():
+            form.save()  # This will create a new Monitor instance
+            messages.success(request, "Form submitted successfully!")
+    else:
+        form = MonitorForm()
+
+    return render(request, "Monitor.html", {"form": form})
```
Comment (arkid15r) [inline website/views.py:4291]: Based on the  current `templates` directory naming convention:
```suggestion
    return render(request, "monitor.html", {"form": form})
```

File: website/views.py (hunk 1 above)
Comment (Sarthak5598) [reply website/views.py:2460]: Ohh! forgot about that thanks

**Review by DonnieBLT — APPROVED** [review]

**Pull request merged**